pxc-57-backup/vault.sh

```diff
@@ -28,8 +28,12 @@ function vault_get() {
     export VAULT_ADDR=$(parse_ini "vault_url" "${keyring_vault}")
     local vault_root=$(parse_ini "secret_mount_point" "${keyring_vault}")/backup
     local gtid=$(parse_ini "galera-gtid" "${sst_info}")
+    if [ "$(grep -e '^[[:space:]]*vault_ca' ${keyring_vault})" ]; then
+        ca_path=$(parse_ini "vault_ca" "${keyring_vault}")
+        ca_opt="--cacert $ca_path"
+    fi
```

Comment:
can it be simplified like this?

```suggestion
    ca_path=$(parse_ini "vault_ca" "${keyring_vault}")
    curl \
        ${ca_path:+--cacert $ca_path} \
```

Reply:
looks complicated